test/lint/lint-python.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+"""
+Check for specified flake8 and mypy warnings in python files.
+"""
+
+import os
+import subprocess
+import sys
+
+DEPS = ['flake8', 'mypy', 'pyzmq']
+MYPY_CACHE_DIR = f"{os.getenv('BASE_ROOT_DIR', '')}/test/.mypy_cache"
```

Comment:
Unrelated: If run outside the CI runner, this will likely result in `/test/.mypy_cache`, which may not be wanted?

Reply:
Yes, I noticed this as well but it is the existing behavior. I used `getenv` with the empty fallback because I wanted to potentially change it to a better one but forgot to think about it. What would you suggest as a better fallback?